test/Confluent.Kafka.Avro.IntegrationTests/test.parameters.json

```diff
@@ -0,0 +1,4 @@
+﻿{
+  "bootstrapServer": "localhost:9092",
```

Comment:
not clear why these are not standard client config properties? (bootstrap.servers,..)

Reply:
the test framework requires them to be the same as the method parameter names in the test methods (so they can't include .'s).